src/ably.ts

```diff
@@ -0,0 +1,20 @@
+import Ably from "ably/promises";
+
+export async function openAblyConnection(authUrl: string, userId: string, channel: string, callback: (req: any) => void, debug?: boolean) {
+  const client = new Ably.Realtime.Promise({
+    authUrl: authUrl,
+    log: { level: debug ? 4 : 1 },
+    authParams: {
+      userId,
+    }
+  });
+  const c = client.channels.get(channel);
```

Comment:
Here you will have to use the rewind parameter to retrieve the last persisted message. https://ably.com/docs/channels/options/rewind